eradiate/scenes/atmosphere/_heterogeneous.py

```diff
@@ -203,19 +190,29 @@ def _sigma_t_fname_validator(self, attribute, value):
     _quantities = {"albedo": "albedo", "sigma_t": "collision_coefficient"}
 
     def __attrs_post_init__(self):
-        # Check for automatic width and height computation
+        # Check attributes consistency
         if self.profile is not None:
             if self.toa_altitude != "auto":
                 raise ValueError(
                     "toa_altitude must be set to 'auto' when profile is set"
                 )
         else:
-            if self.toa_altitude == "auto":
+            if self.albedo_fname is None and self.sigma_t_fname is None:
+                self.toa_altitude = "auto"
+                self.width = "auto"
```

Comment:
Danger zone: This basically means that a user explicitly setting `toa_altitude` or `width` inconsistently will *not* get an error and won't get the object initialised as they expect. I think this should be removed.

Reply:
I completely agree